engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -2626,7 +3733,8 @@ void paladin_t::init_spells()
 
   // spec talents shared among specs
   talents.avenging_wrath_might           = find_talent_spell( talent_tree::SPECIALIZATION, "Avenging Wrath: Might" );
-  talents.relentless_inquisitor          = find_talent_spell( talent_tree::SPECIALIZATION, "Relentless Inquisitor" );
+  talents.relentless_inquisitor = find_talent_spell( talent_tree::SPECIALIZATION, "Relentless Inquisitor" );
+  talents.shield_of_vengeance            = find_spell( 184662 );
```

Comment:
This is going to cause Shield of Vengeance to always cast for Ret, even in talent builds that don't take it. Can we instead initialize it from the talent tree for ret, and initalize it for prot if appropriate?

Reply:
Put Shield of Vengeance back into init_ret with find_talent_spell, it's not needed to always initialize anymore now